dockerize/Dockerfile

```diff
@@ -1,14 +1,14 @@
 FROM centos:7
 EXPOSE 5000
 
-RUN rpm -iUvh http://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-8.noarch.rpm && \
-    yum -y update && \
-    yum -y install wget python34 python34-devel npm supervisor git libffi-devel python-devel libxml2-devel libxml2 libxslt-devel zlib1g-devel libxslt
+RUN rpm -iUvh http://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-8.noarch.rpm
+RUN yum -y update
+RUN yum -y install wget gcc-c++ openssl-devel postgresql-devel python34 python34-devel npm supervisor git libffi-devel python-devel libxml2-devel libxml2 libxslt-devel zlib1g-devel libxslt
 
 WORKDIR /tmp
-RUN wget https://bootstrap.pypa.io/get-pip.py && \
-    python3.4 get-pip.py && \
-    pip3 install --upgrade pip
+RUN wget https://bootstrap.pypa.io/get-pip.py
+RUN python3.4 get-pip.py
```

Comment:
@nharraud there is no need in creating new layers instead keep the `&&` pattern.